tests/test_eload_ingestion.py

```diff
@@ -151,10 +151,23 @@ def test_ingest_all_tasks(self):
                 patch('eva_submission.eload_utils.get_all_results_for_query') as m_get_alias_results, \
                 patch('eva_submission.eload_ingestion.get_vep_and_vep_cache_version') as m_get_vep_versions, \
                 patch('eva_submission.eload_utils.requests.post') as m_post, \
+                patch('eva_submission.eload_ingestion.get_ncbi_assembly_dicts_from_term') as m_get_assembly_dict, \
+                patch('eva_submission.eload_ingestion.retrieve_species_scientific_name_from_tax_id_ncbi') as m_get_scf_name, \
                 patch('eva_submission.eload_ingestion.get_assembly_name_and_taxonomy_id') as m_get_tax, \
                 self._patch_mongo_database():
             m_get_alias_results.return_value = [['alias']]
-            m_get_vep_versions.return_value = (100, 100, 'homo_sapiens')
+            m_get_vep_versions.return_value = (100, 100)
+            m_get_assembly_dict.return_value = [{
+                "taxid": "9606",
+                "assemblyname": "GRCh37.p13",
+                "assemblyaccession": "GCA_000001000.1",
+                "synonym": {
+                    "genbank": "GCA_000001405.14",
+                    "refseq": "GCF_000001405.25",
+                    "similarity": "identical"
+                }
+            }]
+            m_get_scf_name.return_value = 'Home sapiens'
```

Comment:
Typo should be corrected in multiple places...
```suggestion
            m_get_scf_name.return_value = 'Homo sapiens'
```

Reply:
corrected